packages/demo/examples/01-xy-chart/LineSeriesExample.jsx

```diff
@@ -234,46 +243,85 @@ class LineSeriesExample extends React.PureComponent {
-                    showCircle={useVoronoiTrigger || !this.state.programmaticTrigger}
+                <div style={{ marginTop: 36 }}>
+                  <LegendOrdinal
+                    key="legend"
+                    direction="row"
+                    scale={legendScale}
+                    shape={({ fill: style, width, height }) => (
+                      <svg width={width + 12} height={height}>
+                        <line
+                          x1={6}
+                          x2={width + 12}
+                          y1={height / 2 + 1}
+                          y2={height / 2 + 1}
+                          strokeWidth={3}
+                          stroke={style.stroke}
+                          strokeDasharray={style.strokeDasharray}
+                        />
+                      </svg>
+                    )}
+                    fill={({ datum }) => legendScale(datum)}
+                    labelFormat={label => label}
                   />
-                </ResponsiveXYChart>
-              )}
-            </WithTooltip>
-          </div>
+                </div>
+
+                {/* Use WithTooltip to intercept mouse events in stickyTooltip state */}
+                <WithTooltip renderTooltip={this.renderTooltip}>
+                  {({ onMouseLeave, onMouseMove, tooltipData }) => (
+                    <ResponsiveXYChart
+                      ariaLabel="Required label"
+                      eventTrigger={useVoronoiTrigger ? 'voronoi' : 'container'}
+                      eventTriggerRefs={this.eventTriggerRefs}
+                      margin={MARGIN}
+                      onClick={disableMouseEvents ? null : this.handleClick}
+                      onMouseMove={disableMouseEvents || stickyTooltip ? null : onMouseMove}
+                      onMouseLeave={disableMouseEvents || stickyTooltip ? null : onMouseLeave}
+                      renderTooltip={null}
+                      showVoronoi={useVoronoiTrigger}
+                      snapTooltipToDataX
+                      snapTooltipToDataY={useVoronoiTrigger}
+                      tooltipData={tooltipData}
+                      xScale={{ type: 'time' }}
+                      yScale={{ type: 'linear' }}
+                    >
+                      <XAxis label="Time" numTicks={5} />
+                      <YAxis label="Stock price ($)" numTicks={4} />
+                      {seriesProps.map(({ key, ...props }) => (
+                        <LineSeries
+                          key={key}
+                          {...props}
+                          disableMouseEvents={disableMouseEvents}
+                          strokeWidth={2}
+                        />
+                      ))}
+                      <CrossHair
+                        fullHeight
+                        showHorizontalLine={false}
+                        strokeDasharray=""
+                        circleSize={d => (d.y === tooltipData.datum.y ? 6 : 4)}
```

Comment:
Nice Add!   One thing we can possible improve here is to use @vx/glyph to make the shape customizable in the future :)

Reply:
also not a bad idea! that'd change a decent # of props / be a breaking change since a lot of the props have `circle` baked in rn. I wanted to get this in for mdx since they wanted to use this pattern so maybe we can also do that in a future PR.